packages/pipeline/src/Stage.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { forwardRef, ReactNode, Ref, ReactElement } from 'react';
```

Comment:
![Screen Shot 2019-11-18 at 4 58 34 PM](https://user-images.githubusercontent.com/25213299/69097546-c1846080-0a24-11ea-8884-54cc0fd9a3d9.png)

I managed to hit this issue when moving from the `xsmall` size where all stages are visible to the `small` size at which a size would be collapsed. It's my understanding that the squared-off extra stage (or so it seems) shouldn't be there. 

Not sure where exactly someone would hit this issue in practice, but to be safe it's probably worth addressing.

Reply:
Realized it's somewhat difficult to hit this. You can hit this with the resizable container set to `474px`.

Reply:
Nice catch, I wasn't firing the side-effects of the mutation observer for changes to the `childList`. You could also generate the same behaviour by removing childNodes.

I have fixed this in 92c2b15, so you shouldn't run into this anymore in either case:

![pipeline-observer-fix](https://user-images.githubusercontent.com/314505/69303352-bf164680-0c70-11ea-8daf-11ca257250eb.gif)